src/BIP44CoinTypeNode.test.ts

```diff
@@ -648,6 +759,70 @@ describe('getBIP44AddressKeyDeriver', () => {
     expect(deriver(0).toString('base64')).toStrictEqual(expectedKey);
   });
 
+  it('returns the expected BIP-44 address key deriver function (object change value)', () => {
+    const coinType = 60;
+    const parentNode = new BIP44CoinTypeNode([
+      defaultBip39NodeToken,
+      BIP44PurposeNodeToken,
+      `bip32:${coinType}'`,
+    ]);
+    const expectedPath = `m / bip32:44' / bip32:${coinType}' / bip32:0' / bip32:2`;
+
+    const deriver = getBIP44AddressKeyDeriver(parentNode, {
+      change: {
+        index: 2,
+        hardened: false,
+      },
+    });
+    expect(deriver.coin_type).toStrictEqual(coinType);
+    expect(deriver.path).toStrictEqual(expectedPath);
+
+    const expectedKey = new BIP44Node({
+      derivationPath: [
+        defaultBip39NodeToken,
+        BIP44PurposeNodeToken,
+        `bip32:${coinType}'`,
+        `bip32:0'`,
+        `bip32:2`,
+        `bip32:0`,
+      ],
+    }).key;
+
+    expect(deriver(0).toString('base64')).toStrictEqual(expectedKey);
+  });
+
+  it('returns the expected BIP-44 address key deriver function (hardened change value)', () => {
+    const coinType = 60;
+    const parentNode = new BIP44CoinTypeNode([
+      defaultBip39NodeToken,
+      BIP44PurposeNodeToken,
+      `bip32:${coinType}'`,
+    ]);
+    const expectedPath = `m / bip32:44' / bip32:${coinType}' / bip32:0' / bip32:2'`;
+
+    const deriver = getBIP44AddressKeyDeriver(parentNode, {
+      change: {
+        index: 2,
+        hardened: true,
+      },
+    });
+    expect(deriver.coin_type).toStrictEqual(coinType);
+    expect(deriver.path).toStrictEqual(expectedPath);
+
+    const expectedKey = new BIP44Node({
+      derivationPath: [
+        defaultBip39NodeToken,
+        BIP44PurposeNodeToken,
+        `bip32:${coinType}'`,
+        `bip32:0'`,
+        `bip32:2'`,
+        `bip32:0`,
+      ],
+    }).key;
+
+    expect(deriver(0).toString('base64')).toStrictEqual(expectedKey);
```

Comment:
I wonder if we should use something other than `0` for some of these test values? 🤔 

Just to be on the safe side.

Reply:
Added a test for this!